docker-compose.yml

```diff
@@ -7,18 +7,7 @@ services:
     depends_on:
       - mongo
       - redis
-    command:
-      - /wait-for.sh
-      - mongo:27017
-      - '--'
-      - /wait-for.sh
-      - redis:6379
-      - '--'
-      - sh
-      - -c
-      - |
-        mongo "$$OTR_MONGO_URL" --eval 'rs.initiate({ _id: "myapp", members: [{ _id: 0, host: "mongo:27017"}] })'
-        fresh -c scripts/fresh-runner.conf
+    command: tail -f /dev/null
```

Comment:
What's up here? Intended?

Reply:
Accidental commit when trying to re-run travis